dojo/db_migrations/0056_ms_teams.py

```diff
@@ -0,0 +1,117 @@
+from django.db import migrations, models
+import multiselectfield.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0055_notifications_jira_update_verbose_name'),

```

Comment:
needs an update due to new migrations in dev

Reply:
Done